README.md

```diff
@@ -22,21 +22,33 @@ If you know Python, web technologies (JS, HTML, CSS, etc.) or both, make a fork,
 git clone https://github.com/OpenDroneMap/WebODM
 cd WebODM
 pip install docker-compose
-docker-compose -f docker-compose.yml -f docker-compose.nodeodm.yml up
+./webodm.sh start
 ```
 
-* If you're on Windows / OSX, find the IP of your Docker machine by running this command from your Docker Quickstart Terminal:
+* If you're on Windows find the IP of your Docker machine by running this command from your Docker Quickstart Terminal:
 
 ```
 docker-machine ip
```

Comment:
So Linux / Mac uses its own IP address, Windows needs to figure it out via `docker-machine ip`. Was this more a suggestion to change "Linux / Mac" with "{Linux, Mac}"?